includes/rest/class-inbox-controller.php

```diff
@@ -288,4 +278,37 @@ public function get_item_schema() {
 
 		return $this->add_additional_fields_schema( $this->schema );
 	}
+
+	/**
+	 * Extract recipients from the given Activity.
+	 *
+	 * @param array $activity The activity data.
+	 *
+	 * @return array An array of user IDs who are the recipients of the activity.
+	 */
+	private function get_recipients( $activity ) {
```

Comment:
How would you feel about calling it `get_local_recipients()` to reflect the work this function does?